tests/console/docker.pm

```diff
@@ -116,6 +116,21 @@ sub run {
         assert_script_run('man -P cat docker config | grep "docker-config - Manage Docker configs"');
     }
 
+    # Try to stop container using ctrl+c
+    type_string('docker run --rm opensuse/tumbleweed sleep 99999\n');
+    type_string("# Lets's press ctrl+c now ... ");
+    send_key 'ctrl-c';
+    type_string(' ... and we seem to be still in container\n');
+    save_screenshot();
+    # It does not work by purpose - see bsc#10738771 and bsc#1089732
+    # We need to kill the process from another console
+    select_console('user-console');
+    script_sudo('ps xo "pid flag args label" | grep "sleep 99999"');
+    assert_script_sudo('kill -9 $(ps -xo "pid flag args" | egrep "4 sleep 99999$" | cut -d" " -f2)');
+    script_sudo('ps xo "pid flag args label" | grep "sleep 99999"; true');
+    select_console('root-console');
+    assert_script_run('echo "The container was terminated" | grep "The container was terminated"');
```

Comment:
why do you grep here?

Reply:
To ensure the container was terminated ... but maybe the echo itself also returns 0 so I can probably skip the grep part.